src/services/etherspot.js

```diff
@@ -698,6 +699,25 @@ export class EtherspotService {
       return [];
     }
   }
+
+  async getGasPrice(chain: Chain): Promise<?GasPrice> {
+    const sdk = this.getSdkForChain(chain);
+    if (!sdk) return null;
+
+    try {
+      const { standard, fast, instant } = await sdk.getGatewayGasInfo();
+
+      // maps from ethers.js BigNumber
+      return {
+        standard: new BigNumber(standard.toString()),
```

Comment:
Consider using `fromEthersBigNumber`

Reply:
`fromEthersBigNumber` formats wei value into decimal based and wraps it into other BigNumber lib. It doesn't make sense having it here as we want values in wei.